igvm_c/scripts/post_process.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+#
+# Copyright (c) 2024 SUSE LLC
+#
+# Author: Roy Hopkins <roy.hopkins@suse.com>
+#
+# A script to post-process the header files generated with cbindgen
+# to rename items which could not be directly handled by the
+# cbindgen configuration files and annotations.
+set -e
+
+sed -i -e 's/INVALID = 0/IGVM_INVALID = 0/g' \
```

Comment:
I'm not that familiar with sed, these are just transforming the left into the right?

Reply:
Yes